about.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="/styles/styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Poppins:wght@300&display=swap"
+    rel="stylesheet">
+  <title>Mike's Capstone Project</title>
+</head>
+<body>
+  <!-- the navbar (and header) section -->
+  <nav class="mini-nav on-desktop">
+    <ul class="mininav-menu">
+      <li class="mininav-item"><img src="/img/facebook.svg" alt="facebook"></li>
+      <li class="mininav-item"><img src="/img/twitter.svg" alt="twitter"></li>
+      <li class="mininav-menu">English</li>
+      <li class="mininav-menu">My Page</li>
+      <li class="mininav-menu">Logout</li>
+    </ul>
+  </nav>
+  <header id="myheader">
+    <a class=" logo on-desktop" href="#home"><img src="/img/logo.jpg" class="logo-nav" alt="logo"></a>
```

Comment:
- [ ] when in your about page, the logo doesn't take you back to the home page for the desktop version. you can fix that with this `href="/index.html"` or '/'
- [ ] for the mobile menu, kindly add a link that can take you to the home page when in the about page. nice work

Reply:
Dear Sir, I have fixed it. Thanks a lot for pointing it out for me.

Reply:
I have done the changes as suggested. When a user clicks on Program in mobile menu, it takes the user to the home page